src/Css.re

```diff
@@ -428,7 +443,31 @@ let columnCount = x =>
     },
   );
 
-let contentRule = x => D("content", {j|"$x"|j});
+let contentRule = x => D("content", string_of_content(x));
```

Comment:
if possible, I'd like to inline the string_of_content

Reply:
This way there will be duplication as I use the function in `contentRules` as well. What do you propose then?

Reply:
Maybe I could put these functions (together with `string_of_counter_*`) to modules in `AtomicTypes`? I can see you're avoiding this so far (like `string_of_background`), what was the reason of that?

Reply:
just string_of_content. Css.re is already huge, I'm trying to not add useless code. In that case, creating a function doensn't add anything. For operations, it's needed because it's used in multiple locations

Reply:
`string_of_content` is used twice as well. Am I missing something?